### PR TITLE
Update README.md to make contribution steps easier to follow

### PR DESCRIPTION
**Pull request by t1m0thyj**
Update the readme to add installation steps and summary of each step. Thanks @robertblum-psi for suggesting improvements to the readme.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,39 +2,86 @@
...
-        {
-          "commandGroups": ["zos-files", "zos-jobs"],
-          "profiles": ["zosmf", "base"]
-        }
-    *Note: No @zowe scoped plug-ins should be included in this repository, and the above is merely for demonstration purposes.*
+To add your conformant plug-in's commands, perform the following steps:
+> **Note:** All steps should be performed in the root of this project.
+
+1. _Ensure the plug-in you want to contribute to the Web Help is installed in Zowe CLI._<br>
+    If it is not already installed, see step 3 above.
```
Comment (zFernand0) [inline README.md:38]: should we say "See step 3 in the Installing section" or link to it?
Just in case we move some things around in this document.

**Review by zFernand0 — APPROVED** [review]
LGTM! 😋

File: README.md [hunk 2]
```diff
@@ -2,39 +2,86 @@
 
 ## About the Generator
 
-This repository introduces some utilities that can be used to extract the command tree from the CLI and Plug-ins, store the relevant command groups for third party conformant plugins in a legal-approved way (without exposing proprietary code), and generate a web help with those plug-ins included on demand.
+This repository introduces utilities that extract the command tree from the CLI and its installed plug-ins, store the relevant command groups for third party conformant plugins in a legal-approved way (without exposing proprietary code), and generate web help with those plug-ins included on demand.
```
Comment (anaxceron) [inline README.md:5]: hyphenate "plug-in"

File: README.md [hunk 3]
```diff
@@ -2,39 +2,86 @@
...
+    git clone https://github.com/zowe/zowe-cli-web-help-generator.git
+    cd zowe-cli-web-help-generator
+    ```
 
----
+2. Install project dependencies:
+    ```bash
+    npm install --prod
+    ```
+    > **Note:** If you plan to modify the scripts in this repo, omit the `--prod` flag to install dev dependencies as well.
```
Comment (anaxceron) [inline README.md:23]: would it be better to rewrite this?: " ... to also install dev dependencies"

File: README.md [hunk 4]
```diff
@@ -2,39 +2,86 @@
...
 
-## Using - Contributing to the Web Help
+3. Install the plug-in you want to contribute to the Web Help into Zowe CLI, if it is not already installed. If you do not have a globally installed Zowe CLI available, run:
```
Comment (anaxceron) [inline README.md:25]: "web help" is lowercased and uppercased throughout this ReadMe. Don't mix styles. Decide on one style and apply it consistently.

File: README.md [hunk 5]
```diff
@@ -2,39 +2,86 @@
...
-        {
-          "commandGroups": ["zos-files", "zos-jobs"],
-          "profiles": ["zosmf", "base"]
-        }
-    *Note: No @zowe scoped plug-ins should be included in this repository, and the above is merely for demonstration purposes.*
+To add your conformant plug-in's commands, perform the following steps:
+> **Note:** All steps should be performed in the root of this project.
+
+1. _Ensure the plug-in you want to contribute to the Web Help is installed in Zowe CLI._<br>
+    If it is not already installed, see step 3 in the [Installing](#installing) section.
```
Comment (anaxceron) [inline README.md:38]: capitalize Step 3

File: README.md [hunk 6]
```diff
@@ -2,39 +2,86 @@
...
+
+2. _Define command groups and profiles that your plug-in contributes._<br>
```
Comment (anaxceron) [inline README.md:40]: why are these steps italicized? is the ital really necessary?

**Review by anaxceron — APPROVED** [review]
Looks pretty good. 

A few things that you should address:

- Decide whether to use "plug-in" or "plugin." I see both styles in this content, and there should only be one.
- Decide whether to capitalize Web Help or not. I see both styles in this content, and there should only be one.
- Decide whether to italicize your steps. There's no ital in Installing, but there is ital in the other sections. There should only be one formatting style consistently used through this ReadMe.

**Review by gejohnston — APPROVED** [review]

**Review by anaxceron — APPROVED** [review]
Nice job!

**Pull request merged**